composer.json

```diff
@@ -3,10 +3,11 @@
     "type": "symfony-bundle",
     "description": "The missing Symfony Admin Generator",
     "keywords": [
-        "Admin Generator",
         "admin",
+        "bundle",
+        "php",
         "sonata",
-        "bootstrap"
```

Comment:
We are still tied to bootstrap, aren't we? Maybe we should leave this keyword, or replace it with `admin-lte`?

Reply:
Oh, I didn't understand the bootstrap keyword.

I think `admin-lte` is better indeed.